tools/autograd/derivatives.yaml

```diff
@@ -829,9 +829,6 @@
 - name: uniform_(Tensor self, double from, double to, Generator generator)
   self: zeros_like(grad)
 
-- name: _unique(Tensor self, bool sorted, bool return_inverse)
```

Comment:
is there reasons for not implementing autograd for unique?

Reply:
What happens if we didn't put an operator to this list? Isn't `not_implemented` the default? I'm not sure about this. 

I think the only reason for not supporting unique's autograd is, nobody need it, so nobody complain about it, so nobody go ahead and implement it...

Reply:
sure if nobody wants it then we don't need to implement it, but I think it's good to keep the correct function signature here, in your case this might be `unique(Tensor self, bool sorted, bool return_inverse, int64_t? dim)`, and keep the `not_implemented` here because it will be used to give decent error msg to user that this function is not having autograd implemented yet.